src/applications/vaos/services/appointment/transformers.js

```diff
@@ -545,86 +545,105 @@ function setLegacyVAR(appt) {
  * Transforms VAR appointment to FHIR appointment resource
  *
  * @export
- * @param {Array} appointments An array of appointments from the VA facilities api
+ * @param {Object} appt An appointment from MAS or the VAR community care api
+ * @returns {Appointment} An Appointment resource
+ */
+export function transformConfirmedAppointment(appt) {
```

Comment:
No changed logic in these transformers, just reorganizing them so I could call a function to transform a single appointment or request.